internal/cephfs/volume.go

```diff
@@ -128,27 +134,23 @@ type localClusterState struct {
 	subVolumeGroupCreated bool
 }
 
-func createVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID, bytesQuota int64) error {
+func createVolume(ctx context.Context, volOptions *volumeOptions, volID volumeID, bytesQuota int64) error {
 	// verify if corresponding ClusterID key is present in the map,
 	// and if not, initialize with default values(false).
 	if _, keyPresent := clusterAdditionalInfo[volOptions.ClusterID]; !keyPresent {
 		clusterAdditionalInfo[volOptions.ClusterID] = &localClusterState{}
 	}
 
+	ca, err := volOptions.conn.GetFSAdmin()
+	if err != nil {
+		util.ErrorLog(ctx, "could not get FSAdmin, can not create volume %s: %s", volOptions.FsName, err)
```

Comment:
This need to be changed isn't? are we creating a volume in this step?

Reply:
we are creating a volume in the `createVolume()` function, so explaining that it failed would be nice, I think. Most people would not know what `FSAdmin` means, or what the implications are if it can not be obtained.

Reply:
The log will be printed as `can not create volume %s: %s", volOptions.FsName, err` as we are not creating a volume with FsName this would give the wrong impression we are creating only subvolumegroup and subvolume in this case. we need to change the name here it should be string(volID)`

Reply:
Ah, now I understand what you mean :+1: